apps/vgo-cli/src/vgo_cli/repo.py

```
@@ -21,6 +21,20 @@ def load_governance(repo_root: Path) -> dict:
     return load_json(repo_root / 'config' / 'version-governance.json')
 
 
+def _run_git_capture(repo_root: Path, *args: str) -> str:
+    try:
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)